main.go

```diff
@@ -2,21 +2,33 @@ package main
 
 import (
 	"flag"
+	"log"
+
 	"github.com/grubastik/flat-search/aggregator"
 	"github.com/grubastik/flat-search/config"
 	"github.com/grubastik/flat-search/db"
 	"github.com/grubastik/flat-search/email"
-	"github.com/grubastik/flat-search/error"
 )
 
 func main() {
-	config := config.MustNewConfig(*flag.String("config", "./config.json", "Path to the config file"))
-	storage, err := db.NewDb(config)
-	error.DebugError(err)
+	log.Println("start")
+	cfg := config.MustNewConfig(*flag.String("config", "./config.json", "Path to the config file"))
+	storage, err := db.NewDb(cfg)
+	if err != nil {
+		log.Fatal(err, "can't open DB")
+	}
 	defer storage.Close()
 
-	email.NewConnection(config)
+	err = storage.Db.Ping()
+	if err != nil {
+		log.Fatal(err, "ping error")
+	}
+
+	email.NewConnection(cfg)
 
-	err = aggregator.ProcessAdverts(config.GetSreality())
-	error.DebugError(err)
+	err = aggregator.ProcessAdverts(cfg.GetSreality())
+	if err != nil {
+		log.Fatal(err, "process adverts error")
+	}
+	log.Println("end")
```

Comment:
Is all these changes related to migrations?

Reply:
i just wanted to know when and where app stops